tensorflow/cc/framework/cc_op_gen.cc

```diff
@@ -57,6 +57,16 @@ string GetPath(const string& dot_h_fname) {
   return result;
 }
 
+// Converts: some/path/to/file.xx
+// to: file
+// (note that suffix is removed)
+string GetFilename(const string& path) {
```

Comment:
Correct, this affects the comments of the generated header files in the C++ API. So the binary's header files won't have the updated comments, but the docs and manual builds will.

Are the header files even exposed in the binary? (I'm not exactly sure what kind of binary it is... @keveman told me that the headers files aren't included in the pip install, so does that mean they're not exposed?)